### PR TITLE
meow meow

### PR DESCRIPTION
**Pull request by AstorAlina**

### CODE REVIEW

File: Group_Hotel/src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -5,5 +5,6 @@ public static void main(String[] args) {
         System.out.println("Hello world!");
         System.out.println("is this showing?");
         System.out.println("New test");
+        System.out.println("Meow");
```
Comment (herbcerr) [inline Group_Hotel/src/main/java/org/example/Main.java:8]: disappointed

Comment (MCorder00) [inline Group_Hotel/src/main/java/org/example/Main.java:8]: Do you meow how fast you were going?

**Review by MCorder00 — APPROVED** [review]

**Review by herbcerr — APPROVED** [review]

File: Group_Hotel/src/main/java/org/example/Main.java (hunk 1 above)
Comment (AstorAlina) [reply Group_Hotel/src/main/java/org/example/Main.java:8]: .....> . >

Comment (AstorAlina) [reply Group_Hotel/src/main/java/org/example/Main.java:8]: 250meow power

**Pull request merged**